src/rssignores.cpp

```diff
@@ -90,17 +91,34 @@ void RssIgnores::dump_config(std::vector<std::string>& config_output) const
 
 bool RssIgnores::matches(RssItem* item)
 {
+	std::string prefix = "regex:";
```

Comment:
Nitpick: please mark constants with `const`. It makes it easier to read the code (less state to keep in mind) and can help you catch bugs (typos or improper use of values).